config/orm.js

```diff
@@ -0,0 +1,75 @@
+var connection = require("../config/connection.js");
+
+// function to print question marks by how many values are put in, not necessary for this as only burger name is being used
+function printQuestionMarks(num) {
+	var arr = [];
+
+	for (var i = 0; i < num; i++) {
+		arr.push("?");
+	}
+
+	return arr.toString();
+}
+// converts devoured state to mysql so devoured will be changed
+function objToSql(ob) {
+	var arr = [];
+
+  	// loop through the keys and push the key/value as a string int arr
+  	for (var key in ob) {
+  		var value = ob[key];
+    	// check to skip hidden properties
+    	if (Object.hasOwnProperty.call(ob, key)) {
+      		// if string with spaces, add quotations (Lana Del Grey => 'Lana Del Grey')
+      		if (typeof value === "string" && value.indexOf(" ") >= 0) {
+      			value = "'" + value + "'";
+      		}
+	      	// e.g. {name: 'Lana Del Grey'} => ["name='Lana Del Grey'"]
+	      	// e.g. {sleepy: true} => ["sleepy=true"]
+	      	arr.push(key + "=" + value);
+    	}
+  	}
+
+  	// translate array of strings to a single comma-separated string
+  	return arr.toString();
+}
+
+var orm = {
+	// display all of the burgers
+	all: function(cb) {
+		var queryString = "SELECT * FROM burgers;";
+		connection.query(queryString, function(err, result) {
+			if (err) {
+				throw err;
+			}
+			cb(result);
+		});
+	},
+	// when you enter a burger, its entered into mySQL
+	create: function(cols, vals, cb) {
+		var queryString = "INSERT INTO burgers (" + cols.toString() + ") VALUES (" + printQuestionMarks(vals.length) + ")";
+
+		console.log(queryString);
+		console.log("Hey");
```

Comment:
unnecessary console.log that probably shouldn't be pushed into production...Unless there is some lightweight trolling going on